src/main/scala/org/broadinstitute/dsde/firecloud/webservice/SnapshotApiService.scala

```diff
@@ -20,6 +21,11 @@ trait SnapshotApiService extends FireCloudDirectives with UserInfoDirectives {
         pathEnd {
           post {
             passthrough(SnapshotApiService.createDataRepoSnapshotURL(namespace, name), HttpMethods.POST)
+          } ~
+          get {
+            parameters("offset", "limit") { (offset, limit) =>
+              passthrough(SnapshotApiService.listDataRepoSnapshotURL(namespace, name, offset, limit), HttpMethods.GET)
+            }
```

Comment:
for passthrough apis with a querystring, we typically handle it by passing the querystring as-is. See https://github.com/broadinstitute/firecloud-orchestration/blob/develop/src/main/scala/org/broadinstitute/dsde/firecloud/webservice/WorkspaceApiService.scala#L61 for an example.

Your solution would work too; this is more about consistency in the codebase. If there is a reason to extract/verify only specific params, I'd use your solution; passing it through untouched means we can add/change params in Rawls without needing changes here in orch.

Reply:
SGTM - it's better to pass it through untouched in this case

Reply:
this should not pass through to `FireCloudConfig.Rawls.baseUrl` - it should pass through to `listDataRepoSnapshotURL(namespace, name)`, and `listDataRepoSnapshotURL` should be ignorant of any querystring.